Dockerfile

```diff
@@ -10,11 +9,12 @@ RUN apk add git musl-dev curl
 ARG SHELL=/bin/ash
 RUN curl --proto '=https' --tlsv1.2 -sSf https://docs.swmansion.com/scarb/install.sh | $SHELL -s -- -v $SCARB_VERSION
 
-# Install cairo1 compiler
-RUN git clone -b $CAIRO_COMPILER_TARGET_TAG https://github.com/starkware-libs/cairo.git
+ARG COMPILER_BINARY_URL
```

Comment:
IMO it's an unnecessary env var (i.e. build arg)

Reply:
Kept it as discussed.